libcloud/compute/drivers/gce.py

```diff
@@ -1456,6 +1456,26 @@ def set_named_ports(self, named_ports):
         return self.driver.ex_instancegroup_set_named_ports(
             instancegroup=self.instance_group, named_ports=named_ports)
 
+    def set_autoHealingPolicies(self, healthcheck, initialdelaysec):
```

Comment:
For consistency, please use camel_case naming - ``set_autoHealingPolicies`` -> ``set_auto_healing_policies``.

Reply:
Same thing for method arguments: healthcheck -> health_check, initdelaysec -> init_delay_sec.

Reply:
I just noticed some other methods also use this notation.

It's not ideal, but for consistency reasons I'm fine with using names as it is.